src/main/java/io/github/dsse/helpers/SimpleECDSASigner.java

```diff
@@ -0,0 +1,30 @@
+package io.github.dsse.helpers;
+
+import io.github.dsse.models.Signer;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.Signature;
+import java.security.SignatureException;
+
+public class SimpleECDSASigner implements Signer {
+  private final PrivateKey privateKey;
+
+  public SimpleECDSASigner(PrivateKey privateKey) {
+    this.privateKey = privateKey;
+  }
+
+  @Override
+  public byte[] sign(String payload)
+      throws NoSuchAlgorithmException, InvalidKeyException, SignatureException {
+    Signature signature = Signature.getInstance("SHA1withECDSA");
+    signature.initSign(privateKey);
+    signature.update(payload.getBytes());
+    return signature.sign();
+  }
+
+  @Override
+  public String getKeyId() {
```

Comment:
instead of returning 'null' could you add a keyid parameter to the constructor?

We plan to use the keyid hint pretty heavily and it would be a bummer if people didn't usually set it.